services/web/client/source/class/osparc/dashboard/StudyBrowser.js

```diff
@@ -628,6 +639,40 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
       }
     },
 
+    __cloneStudy: function(studyData) {
+      const cloneTask = new osparc.component.task.Clone(studyData);
+      cloneTask.start();
+      const text = this.tr("Cloning process started and added to the background tasks");
+      osparc.component.message.FlashMessenger.getInstance().logAs(text, "INFO");
+
+      const cloningStudyCard = new osparc.dashboard.StudyBrowserButtonPlaceholder();
+      cloningStudyCard.buildLayout(
+        this.tr("Cloning ") + studyData["name"],
+        "@FontAwesome5Solid/copy/60"
+      );
+      cloningStudyCard.subscribeToFilterGroup("sideSearchFilter");
+      this.__userStudyContainer.addAt(cloningStudyCard, 1);
+
+      const params = {
+        url: {
+          projectId: studyData["uuid"]
+        },
+        data: osparc.utils.Utils.getClientSessionID()
+      };
+      osparc.data.Resources.fetch("studies", "duplicate", params)
```

Comment:
why are you calling it ``duplicate`` in the API and ``clone`` in the UI? At this point we are still in time to call things in one way :-) ... or we will end up again with the study/project/node **unnecessary** dilemmas. Please select the one you like the most but only one.

Reply:
```duplicate``` it is, then!